imap_processing/tests/ialirt/unit/test_process_ephemeris.py

```diff
@@ -0,0 +1,96 @@
+"""Test processEphemeris functions."""
+
+import numpy as np
+import pytest
+
+from imap_processing.ialirt import process_ephemeris
+from imap_processing.spice.time import str_to_et
+
+
+def test_calculate_doppler():
+    """
+    Test the calculate_doppler() function.
+    """
+
+    assert process_ephemeris.calculate_doppler() == 1
+
+
+@pytest.mark.external_kernel()
+def test_latitude_longitude_to_ecef(furnish_kernels):
+    """
+    Test the latitude_longitude_to_ecef() function.
+
+    Test data is from https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/C/cspice/
+    georec_c.html.
+    """
+    longitude = 118.0  # degrees
+    latitude = 30.0  # degrees
+    altitude = 0.0  # kilometers
+
+    kernels = ["pck00011.tpc"]
+    with furnish_kernels(kernels):
+        rect_coords = process_ephemeris.latitude_longitude_to_ecef(
+            longitude, latitude, altitude
+        )
+
+    assert np.round(rect_coords[0], 6) == -2595.359123
+    assert np.round(rect_coords[1], 6) == 4881.160589
+    assert np.round(rect_coords[2], 6) == 3170.373523
```

Comment:
Using `np.testing.assert_allclose()` for comparison is a great way to not have to implement rounding. The benefit is that the output is really nice when it fails.
```suggestion
    np.testing.assert_allclose(rect_coords, [-2595.359123, 4881.160589, 3170.373523], atol=1e-6)
```